backend/routers/todoRouters.js

```diff
@@ -0,0 +1,76 @@
+import { Router } from "express";
+import { getDataBase } from "../db/index.js";
+import { validateValue } from "../helpers.js";
+
+const db = getDataBase();
+const router = new Router();
+
+// Получение всех задач
+router.get("/", async (req, res) => {
+  try {
+    const allTodos = await db.findAsync({});
+    res.send(allTodos);
+  } catch (error) {
+    res.status(500).send({ error: "Failed to receive tasks" });
+  }
+});
+
+// Создание новой задачи
+router.post("/", async (req, res) => {
+  const body = req.body;
+
+  try {
+    await validateValue(body);
+    const task = await db.insertAsync({
+      text: body.text,
+      is_checked: body.is_checked,
+    });
```

Comment:
As I commented before, the validate function returns the valid object that you can use
```suggestion
    const body = await validateValue(req.body);
    const task = await db.insertAsync(body);
```
should do the trick